docs/configuration.md

```diff
@@ -714,6 +714,19 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.executor.cores</code></td>
+  <td>(infinite)</td>
+  <td>
+    Default: 1 in YARN mode, all the available cores on the worker in standalone mode.
```

Comment:
When I wrote this, I meant to put this phrase into L718, which is really the default column:
```
<td><code>spark.executor.cores</code></td>
<td>1 in YARN mode, all the available cores on the worker in standalone mode.</td>
<td>
  ...
</td>
```